wpilibcExamples/src/main/cpp/examples/XRPReference/cpp/subsystems/Arm.cpp

```diff
@@ -8,6 +8,6 @@ void Arm::Periodic() {
   // This method will be called once per scheduler run.
 }
 
-void Arm::SetAngle(double angleDeg) {
-  m_armServo.SetAngle(angleDeg);
+void Arm::SetAngle(units::radian_t angleRad) {
+  m_armServo.SetAngle(angleRad);
```

Comment:
Unit suffixes aren't needed in C++ since the unit is encoded in the type system.
```suggestion
void Arm::SetAngle(units::radian_t angle) {
  m_armServo.SetAngle(angle);
```